summerofcode/2023.md

```diff
@@ -26,12 +26,26 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+- [DevStream](#devstream)
```

Comment:
let's not add the ToC entry at this point, we can add it at the end of the process. Just the project idea is enough for now.